images/win/scripts/Installers/Install-Msys2.ps1

```diff
@@ -0,0 +1,80 @@
+################################################################################
+##  File:  Install-Msys2.ps1
+##  Desc:  Install Msys2 and 64-bit gcc, cmake, & llvm (32-bit commented out)
+################################################################################
+
+# References
+# https://github.com/msys2/MINGW-packages/blob/master/azure-pipelines.yml
+# https://packages.msys2.org/group/
+
+$env:origPath = $env:PATH
```

Comment:
I think it's not necessary to use $env at all: 
$env:origPath => $origPath 
$env:gitPath = $gitPath

Reply:
Done